### PR TITLE
Feedback prac05

### PR DESCRIPTION
**Pull request by Blake0982**
Prac 5 code review request @MichaelHowell1
review code for errors and possible improvements, only had time to finish 4/5 codes
have a good one 👍

### CODE REVIEW

File: prac_05/email.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
+"""
+Prac 5
```
Comment (MichaelHowell1) [inline prac_05/email.py:2]: General note for all function/programs perhaps adding docstrings with brief summary at the start of the code and immediately below functions would help explain each section of code.

File: prac_05/email.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+"""
+
+
+def main():
+    global fullname
```
Comment (MichaelHowell1) [inline prac_05/email.py:7]: I am pretty sure Lindsay doesn't like global variables perhaps an alternative way should be used.

File: prac_05/email.py [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    name = name.split(".")
+    try:
+        fullname = name[0] + " " + name[1]
+        fullname = fullname.title()
+        print(f"Is your name: {fullname}?")
+    except IndexError:
+        print(f"Is your name: {name}?")
+    answer = input("Y/N: ").upper()
+    if answer.startswith("N"):
+        fullname = input("Name:")
```
Comment (MichaelHowell1) [inline prac_05/email.py:30]: Can this return fullname without the use of a global variable?

File: prac_05/hex_colours.py [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+COLOUR_TO_CODES = {"aliceblue": "#f0f8ff", "antiquewhite": "#faebd7",
+                   "antiquewhite1": "#ffefdb", "antiquewhite2": "#eedfcc",
+                   "antiquewhite3": "#cdc0b0", "antiquewhite4": "#8b8378",
+                   "aquamarine1": "#7fffd4", "aquamarine2": "#76eec6",
+                   "aquamarine4": "#458b74", "azure1": "#f0ffff",
+                   "azure2": "#e0eeee", "azure3": "#c1cdcd", "azure4": "#838b8b",
+                   "beige": "#f5f5dc", "bisque1": "#ffe4c4"}
+print("Hex Colours, type 'Quit' to exit!")
+colour_name = input("Enter a colour name: ").lower()
+while colour_name != "Quit":
```
Comment (MichaelHowell1) [inline prac_05/hex_colours.py:14]: May not matter too much but practical asked for "Allow the user to enter names until they enter a blank one to stop the loop." you have "Quit". Still works obviously but Lindsay may take off marks.

**Review by MichaelHowell1 — COMMENTED** [review]
Just a couple of small changes that may help boost marks particularly docstrings but overall code is good.

File: prac_05/email.py (hunk 2 above)
Comment (lindsaymarkward) [reply prac_05/email.py:7]: It's not just Lindsay :)

**Pull request merged**